examples/geoflow/imidazole.in

```diff
@@ -2,13 +2,14 @@ read
     mol pqr imidazole_zap.pqr
 end
 elec name comp_solv        # Solvated complex
-    geoflow
+    geoflow-auto
 
     lpbe
     bcfl mdh                # only supported boundary condition
     mol 1
 
-    grid 0.25 0.25 0.25     # grid spacing                    
+    # WARN: grid is not longer supported
+    # grid 0.25 0.25 0.25     # grid spacing                    
```

Comment:
Is this really not supported or does it just produce a warning about it being deprecated?

Reply:
I don't know. The answer will be the same as the previous comment about `grid` being part of the `geoflow-auto` spec or not.

Reply:
That's odd.  Does the example run successfully?